Sources/Chat/ProtocolServices/Common/MessagingService.swift

```diff
@@ -4,9 +4,8 @@ import Combine
 class MessagingService {
 
     private let keyserverURL: URL
-    private let networkingInteractor: NetworkInteracting
-    private let identityStorage: IdentityStorage
-    private let identityService: IdentityService
+    private let networkClient: NetworkInteracting
```

Comment:
should we rename networkingInteractor to networkClient? we already got use to `networkingInteractor` and it's consistent in all the SDKs

I personally wouldn't do it

Reply:
reverted, planning to discuss it later